.changeset/tough-bobcats-retire.md

```diff
@@ -2,4 +2,6 @@
 "fuels-wallet": minor
 ---
 
-Allow users to switch to an existing network through the `addNetwork` flow, even if the network already exists in their wallet.
\ No newline at end of file
+Allow users to switch to an existing network through the `selectNetwork` flow.
+It creates the network if it doesn't exist yet.
+It selects the network if it already exists in their wallet.
```

Comment:
I'm wondering why you suggest that?

I'm referencing this current change (which is "allowing users to switch network"), those lines you mentioned are not the change itself – but an explanation.

```
Allow users to switch to an existing network through the `selectNetwork` flow.
It creates the network if it doesn't exist yet.
It selects the network if it already exists in their wallet.
```

Reply:
because it will look ugly in the changelog. It should be a single sentence then if you want to include all this info, maybe you can reword it to make it smaller

Reply:
we should not break lines in changeset